video_xblock/static/bower.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "xblock-video",
+  "version": "0.2-dev",
+  "dependencies": {
+    "video-js": "5.13.1",
+    "videojs-youtube": "2.1.1",
+    "videojs-wistia-extra": "*"
```

Comment:
Can you move this fork from https://github.com/dorosh/videojs-wistia.git to raccoongang repo